source/incident-collaboration/playbook-planning.rst

```diff
@@ -1,47 +1,67 @@
-Setting Up Playbooks
+Planning a Playbook
 ====================
 
-A playbook must be defined before starting an incident.
+Playbooks are made up of:
 
-Adding tasks
-~~~~~~~~~~~~~
+- Checklists: The list of tasks to be completed for the run.
+- Templates: Templates for frequently-used actions such as updates and reminders. 
+- Actions: Automation options for inviting members, webhooks, welcome messages and more.
+- Permissions: Manage permissions at a channel and a playbook level.
+
+Creating checklists
+-------------------
 
 1. Go to **Main Menu > Incident Collaboration**.
 2. Select **Playbooks**.
-3. Start a **Blank Playbook**, or use the built-in **Incident Collaboration Playbook** as a template.
+3. Start a **Blank Playbook**, or use the built-in template.
 4. Name your playbook and provide a description.
-5. Select the **Tasks** tab.
-6. Within the **Tasks** tab, customize the checklists and tasks:
-  * Create and name new tasks to capture actions your team should take to resolve the incident.
-  * Create new checklists to group tasks meant to be completed together.
-  * Drag and drop to reorganize checklists and tasks.
-  * Optionally add task descriptions to give additional context to members of the incident channel. Descriptions support a limited form of Markdown, including text styling and hyperlinks.
-  * Optionally add a slash command to the task that can be executed by members of the incident channel as needed.
+5. Select the **Checklists** tab.
+
+You can start with the default checklist and edit it or you can delete it and select **+ New checklist**. Each new section of a checklist is called a stage.
```

Comment:
Is this the terminology we're using now? I honeslty don't know, but I thought it was `task` for each checkbox and `checklist` for the collection of tasks.

Reply:
I wrote this based on the UI which tbh did confuse me so I'm glad you raised this. However, I just looked on daily now and it doesn't reference stage at all so this will be updated!! Good catch.